examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,581 @@
+plt.plot(xp, yp, marker='o')
+
+# Annotation settings
+plt.annotate(r'${(\sigma_{yp})_s}$', xy=(xp, yp), xytext=(0.05, 75000),
+             arrowprops=dict(facecolor='steelblue', shrink=0.05),
+             bbox=dict(facecolor='steelblue', edgecolor='black', boxstyle='round, pad=1'))
+
+# Define stress - strain properties of the aluminum.
+aluminum = {"stress_a": [0, 55000, 55000, 55000],
+            "strain_a": [0, 0.05, 0.1, 0.2]}
+
+# Define yielding strength point of the Aluminum on the curve.
+xp = aluminum["strain_a"][1]
+yp = aluminum["stress_a"][1]
+
+# Set up the settings of the aluminum curve.
+plt.plot(aluminum["strain_a"], aluminum["stress_a"],
+         label="2024-T4 Aluminum", linewidth=2,
+         color='sandybrown', linestyle="-", marker="o")
+plt.plot(xp, yp, marker='o')
+
+# Annotation settings
+plt.annotate(r'${(\sigma_{yp})_a}$', xy=(xp, yp), xytext=(0.07, 45000),
+             arrowprops=dict(facecolor='sandybrown', shrink=0.05),
+             bbox=dict(facecolor='sandybrown', edgecolor='black', boxstyle='round, pad=1'))
+
+plt.grid(True)
+plt.legend()
+plt.title("Stress - Strain Curve", fontsize=18)
+plt.show()
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell and pipe elements.
+
+# Shell cross-section for inside (STEEL) tube.
+mapdl.sectype(1, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 1, 0, 5)
+
+# Shell cross-section for outside (Aluminum) tube.
+mapdl.sectype(2, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 2, 0, 5)
+
+# Define Pipe cross-section for inside (STEEL) tube, where:
+mapdl.sectype(3, "PIPE")
+
+# Outside diameter and wall thickness settings for Inside Tube (PIPE288)
+mapdl.secdata(4.9563384, 0.5)
+
+# Pipe cross-section for outside (Aluminum) tube.
+mapdl.sectype(4, "PIPE")
+
+# Outside diameter and wall thickness settings for Outside Tube (PIPE288)
+mapdl.secdata(8.139437, 0.5)
+
+# Print the section properties for all sections.
+print(mapdl.slist())
+
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the nodes and create the elements through the nodes.
+
+# Generate nodes and elements for PIPE288.
+mapdl.n(1, x=0, y=0, z=0)
+mapdl.n(2, x=0, y=0, z=10)
+
+# Create element for steel(inside) tube cross-section.
+mapdl.mat(1)
+mapdl.secnum(3)
+mapdl.e(1, 2)
+
+# Create element for aluminum(outside) tube cross-section.
+mapdl.mat(2)
+mapdl.secnum(4)
+mapdl.e(1, 2)
+
+# Activate the global cylindrical coordinate system.
+mapdl.csys(1)
+
+# Generate nodes and elements for SOLID185.
+mapdl.n(node=101, x=1.9781692)
+mapdl.n(node=101, x=1.9781692)
+mapdl.n(node=102, x=2.4781692)
+mapdl.n(node=103, x=3.5697185)
+mapdl.n(node=104, x=4.0697185)
+mapdl.n(node=105, x=1.9781692, z=10)
+mapdl.n(node=106, x=2.4781692, z=10)
+mapdl.n(node=107, x=3.5697185, z=10)
+mapdl.n(node=108, x=4.0697185, z=10)
```

Comment:
You could further reduce (if you wish so)

```suggestion
nodes_x = [1.9781692, 1.9781692, 2.4781692, 3.5697185, 4.0697185, 1.9781692, 2.4781692, 3.5697185, 4.0697185]
for i, each_x in enumerate(nodes_x):
    mapdl.n(node=100+i, x=each_x, z= 10 if 100+i >104 else 0)
```

Reply:
It is a sparkling idea, I thought about that. 
But then I have thought not to do this simple code in such a complicated way
with "for" loop and "if-else" statements". 

I am really happy that I am thinking in the similar way as more experienced colleague as you)
Thank you, German(@germa89).